astropy/nddata/mixins/ndarithmetic.py

```diff
@@ -8,10 +8,16 @@
 
 import numpy as np
 
-from ... units import dimensionless_unscaled, UnitsError
+from ...units import dimensionless_unscaled, Quantity
```

Comment:
@wkerzendorf @embray I'll stick with the relative imports for now. If it shows that there is a downside to it or isn't considered good coding style I'll change them.